dvc/command/status.py

```diff
@@ -11,6 +11,7 @@ class CmdDataStatus(CmdDataBase):
     STATUS_LEN = 20
     STATUS_INDENT = "\t"
     UP_TO_DATE_MSG = "Data and pipelines are up to date."
+    IN_SYNC_MSG = "Cache and remote '{remote}' are in sync."
```

Comment:
There could be a better phrased message, I'm open to suggestions.

Reply:
Hm, `Cache and remote are in sync` sounds a bit like cache and remote have the same list of cache files, which is actually not true, because we care about the cache for the local workspace or branches/tags (--all-tags, --all-branches). But this is still a good start :+1:

Reply:
I used this paragraph from `dvc status` as a guide for what is happening and how it should be called:
> For new and deleted data, the cache is different from remote storage. Bringing the two into sync requires dvc pull or dvc push. For the typical process to update the workspace, see Sharing Data And Model Files.

Reply:
So if you don't need to do anything to bring them into sync I thought we can call the state as "are in sync" :)

Reply:
I'm keen to work out the best message cause this PR is all about making the message more accurate. Having an OK message kind of defeats the purpose...

Reply:
@tenitski Thanks for the attention to details! :pray: 

@jorgeorpinel could you take a look, please? :pray:

Reply:
Minor: based on the `elif` order below wouldn't it be better to put this new constant before `UP_TO_DATE_MSG`?
As for the text itself, it seems good but mentioning the cache seems too low-level. Maybe "This repo is in sync with the remote." ? Also, may be no need to specify which remote since the user should know that already... But I guess for when you have a default remote, to make sure you're aware of which one was checked, appending the remote name at the end is not a bad idea.